app/helpers/hosts_helper.rb

```diff
@@ -304,14 +304,6 @@ def has_dhcp_lease_errors?(errors)
     errors.include?(:dhcp_lease_error)
   end
 
-  def args_for_compute_resource_partial(host)
-    args = {}
-    args[:arch] = host.try(:architecture_id) || (params[:host] && params[:host][:architecture_id])
-    args[:os] = host.try(:operatingsystem_id) || (params[:host] && params[:host][:operatingsystem_id])
-    args[:selected_cluster] = vm_attrs['cluster'] if defined?(vm_attrs)
-    args
-  end
```

Comment:
FYI: This change broke the proxmox plugin (see https://github.com/theforeman/foreman_fog_proxmox/issues/199) and potentially others. In the future, please make sure to deprecate methods before removing them completely.